.github/workflows/ci.yml

```
@@ -93,6 +93,37 @@ jobs:
             fi
           # Each line of install_manifest.txt contains the location of an installed library or header
           done <install_manifest.txt
+          # Uninstall and reset build.
+          sudo xargs rm < install_manifest.txt || echo "transport_interface.h is installed twice because of duplicate copies in spoke repos."
+          cd .. && rm -rf build
+      - name: Configure CMake build for default install location and selective installation
+        run: |
+          curl https://cmake.org/files/v3.2/cmake-3.2.0-Linux-x86_64.tar.gz -o cmake.tar.gz
+          tar -xf cmake.tar.gz
+          mkdir build && cd build
+          ../cmake-3.2.0-Linux-x86_64/bin/cmake .. \
+          -DBUILD_DEMOS=0 -DBUILD_TESTS=0 \
+          -DINSTALL_LIBS="http;defender" -DINSTALL_PLATFORM_ABSTRACTIONS=0
+      - name: Install libraries and verify default install location
+        run: |
+          cd build && sudo make install
+          while IFS="" read -r p || [ -n "$p" ]
+          do
+            if [[ $p != "/usr/local/include"* ]] && [[ $p != "/usr/local/lib"* ]]; then
+              exit 1
+            fi
+          # Each line of install_manifest.txt contains the location of an installed library or header
+          done <install_manifest.txt
+      - name: Verify filenames from selective build
+        run: |
+          cd build && sudo make install
+          while IFS="" read -r p || [ -n "$p" ]
+          do
+            if [[ $p != *"mqtt"* ]] && [[ $p != *"defender"* ]]; then
```

Should be checked for HTTP which is passed in `INSTALL_LIBS`
```suggestion
            if [[ $p != *"http"* ]] && [[ $p != *"defender"* ]]; then
```